lib/solidus_stripe/engine.rb

```diff
@@ -18,11 +18,13 @@ class Engine < Rails::Engine
 
     initializer "solidus_stripe.pub_sub", after: "spree.core.pub_sub" do |app|
       require "solidus_stripe/webhook/event"
+
+      SolidusStripe::Webhook::Event.register(
+        user_events: SolidusStripe.configuration.webhook_events,
+        bus: Spree::Bus
+      )
+
       app.reloader.to_prepare do
-        SolidusStripe::Webhook::Event.register(
-          user_events: SolidusStripe.configuration.webhook_events,
-          bus: Spree::Bus
-        )
         SolidusStripe::Webhook::PaymentIntentSubscriber.new.subscribe_to(Spree::Bus)
```

Comment:
Did you investigate if we can also move the subscribers out of the `to_prepare` block? I'm not sure but maybe we are subscribing to events multiple times if we keep them in?